test/e2e/specs/importer/importer__import-focused.ts

```diff
@@ -0,0 +1,143 @@
+	const credentials = {
+		simpleSiteFreePlanUser: SecretsManager.secrets.testAccounts.simpleSiteFreePlanUser,
+		jetpackRemoteSiteUser: SecretsManager.secrets.testAccounts.jetpackRemoteSiteUser,
+	};
+
+	let page: Page;
+	let startImportFlow: StartImportFlow;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+		startImportFlow = new StartImportFlow( page );
+	} );
+
+	const loginAsUser = ( user: TestAccountName ) => {
+		it( `Login as ${ user }`, async () => {
+			const testAccount = new TestAccount( user );
+			await testAccount.authenticate( page );
+		} );
+	};
+
+	const navigateToImportFocusedFlow = ( step: string, siteSlug?: string, from?: string ) => {
+		it( `Navigate to page setup/import-focused?from=${ from }`, async () => {
+			await startImportFlow.startImportFocused( step, siteSlug ?? '', from ?? '' );
+		} );
+	};
+
+	describe( 'Scenario: from `self-hosted` (without JP) to `simple` site', () => {
+		const user = 'simpleSiteFreePlanUser';
+
+		loginAsUser( user );
+		navigateToImportFocusedFlow( 'import', credentials[ user ].testSites?.primary?.url as string );
+
+		it( 'Enter make.wordpress.org', async () => {
+			await startImportFlow.enterURL( 'make.wordpress.org' );
+		} );
+
+		it( 'Should render "Upgrade Plan" screen', async () => {
+			await startImportFlow.validateUpgradePlanPage();
+		} );
+
+		it( 'Should render "Install Jetpack" screen', async () => {
+			await startImportFlow.clickButton( 'Upgrade and migrate' );
+			await startImportFlow.validateInstallJetpackPage();
+		} );
+	} );
+
+	describe( 'Scenario: from `self-hosted` (JP connected) to `simple` site', () => {
+		const user = 'jetpackRemoteSiteUser';
+		const sourceSiteUrl = credentials[ user ]?.testSites?.primary?.url as string;
+		const targetSiteUrl = 'e2eimportflowtesting.wordpress.com';
+
+		loginAsUser( user );
+		navigateToImportFocusedFlow( 'import', targetSiteUrl );
+
+		it( `Enter jetpack connected source site: ${ sourceSiteUrl }`, async () => {
+			await startImportFlow.enterURL( sourceSiteUrl );
+		} );
+
+		it( 'Should render "Upgrade Plan" screen', async () => {
+			await startImportFlow.validateUpgradePlanPage();
+		} );
+
+		it( 'Should redirect to "Checkout" page', async () => {
+			await startImportFlow.clickButton( 'Upgrade and migrate' );
+			await startImportFlow.validateCheckoutPage();
+		} );
+	} );
+
+	describe( 'Scenario: from `self-hosted` (JP connected) to `atomic` site', () => {
+		const user = 'jetpackRemoteSiteUser';
+		const sourceSiteUrl = credentials[ user ]?.testSites?.primary?.url as string;
+		const targetSiteUrl = 'e2eimportflowatomictesting.wordpress.com';
+
+		loginAsUser( user );
+		navigateToImportFocusedFlow( 'import', targetSiteUrl );
+
+		it( `Enter jetpack connected source site: ${ sourceSiteUrl }`, async () => {
+			await startImportFlow.enterURL( sourceSiteUrl );
+		} );
+
+		it( 'Should render "Upgrade Plan" screen', async () => {
```

Comment:
This probably needs to say render "Migration ready screen"?

Reply:
Good catch! It was a copy -> paste issue. 🤷‍♂️ 

Updated!